['Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpl86fiswn/solution_0.py", line 62, in <module>\n    print(Solution().countOfPairs(n = 3, x = 1, y = 3))\nTypeError: countOfPairs() got an unexpected keyword argument \'n\'\n', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpl86fiswn/solution_1.py", line 62, in <module>\n    print(Solution().countOfPairs(n = 5, x = 2, y = 4))\nTypeError: countOfPairs() got an unexpected keyword argument \'n\'\n']